### PR TITLE
Convert "Const, Immutable and Shared" to a SECTION2

### PR DESCRIPTION
**Pull request by wilzbach**
See also: http://dtest.dlang.io/artifact/website-e6672ddd5f4efc6e60942fd38836a7bb2abc1361-6511322dd0ebeb894df6eea5bc0109e9/web/spec/class.html

image

From https://github.com/dlang/dlang.org/pull/2043

### CODE REVIEW

File: spec/class.dd [hunk 1]
```diff
@@ -1317,7 +1317,7 @@ $(D new) $(D $(LPAREN))$(I ArgumentList)$(D $(RPAREN)) $(I Identifier) $(D $(LPA
 
 where $(I Identifier) is the name generated for the anonymous nested class.
 
-$(SECTION3 $(LEGACY_LNAME2 ConstClass, const-class, Const, Immutable and Shared Classes),
+$(SECTION2 $(LEGACY_LNAME2 ConstClass, const-class, Const, Immutable and Shared Classes),
```
Comment (andralex) [inline spec/class.dd]: The doc is weird, only shows "Const". I think you need: `$(SECTION2 $(LEGACY_LNAME2 ConstClass, const-class, $(ARGS Const, Immutable and Shared Classes)),`

**Comment by wilzbach** [comment]
> The doc is weird,

Yeah this is still a small issue with the TOC generator, but that's unrelated to this PR.
`SECTION2` works like `SECTION3`:

image

**Comment by andralex** [comment]
@wilzbach I figure but can you please do both improvements here? it's quick and easy.

**Comment by wilzbach** [comment]
> @wilzbach I figure but can you please do both improvements here? it's quick and easy.

Done.

**Pull request merged**